docs/apache-airflow/core-concepts/executor/debug.rst

```diff
@@ -15,8 +15,47 @@
     specific language governing permissions and limitations
     under the License.
 
+.. _executor:DebugExecutor:
```

Comment:
No changes here other than bringing this section to the top to satisfy the static checks we have.